static/js/components/PersonalTab.js

```diff
@@ -68,7 +68,15 @@ export default class PersonalTab extends React.Component {
     dispatch(setProfileStep(PERSONAL_STEP));
   }
 
-  onChange = (selection: Option): void => {
+  componentDidUpdate() {
+    const { currentProgramEnrollment } = this.props;
+    const selectedProgram = this.getSelectedProgramId();
+    if ( currentProgramEnrollment && !selectedProgram ) {
+      this.onProgramSelect({ value: currentProgramEnrollment.id, label: '' });
```

Comment:
this smells a bit to me. we're directly calling a function that is intended to handle the `onChange` event for a select box? shouldn't we just call `setProgram` directly?